pkg/bufferdecoder/eventsreader.go

```diff
@@ -149,7 +149,8 @@ func ReadArgFromBuff(ebpfMsgDecoder *EbpfDecoder, params []trace.ArgMeta) (trace
 		if err != nil {
 			return argMeta, nil, fmt.Errorf("error reading byte array size: %v", err)
 		}
-		if size > 4096 {
+		// error if byte buffer is too big (and not a network event)
+		if size > 4096 && argMeta.Name != "payload" {
```

Comment:
How can we be sure `payload` will only ever be used in net events? It's a rather generic name and I can see it being used in other events in the future (especially when we add plugins and users can add their own events).

Reply:
We can't, but I did not want to pass the eventID as an argument for the function. I could rename the payload arg to something that would only be used by networking code as well OR we can change the function prototype, I'm good with either.

Reply:
Maybe we should mark the argument type as something else? like `NetworkBytes`, then we can have a special handler for that.
@yanivagman WDYT?

Reply:
Okay, I have found a way to deal with this, but in the next PR once I remove the old network events and move the new ones to the "network event ID range", then I'll pass an argument to ReadArgFromBuff with eventID and check if we are in the network event range before checking this limit. For now, keep this as is with a TODO for me to do in another PR in this same milestone.